Makefile.in

```diff
@@ -3956,6 +3956,7 @@ COND_USE_GUI_1_ALL_GUI_HEADERS =  \
 	wx/generic/dvrenderer.h \
 	wx/calctrl.h \
 	wx/propdlg.h \
+	wx//memoryallocator.h \
```

Comment:
I know nothing about makefiles but those doubled slashes look odd? There's few more of those in other places.

Reply:
@PBfordev 
I don't see any double slashes.
The backslashes are standard for the string continuation. It indicates that the next line is a continuation of the current one. I.e., this piece is translated normally to:

wx/generic/dvrenderer.h wx/calctrl.h wx/propdlg.h wx/memoryallocator.h

but in this case the line becomes very long. So in order to break the line the syntax above is used.

Reply:
I was not talking about backslashes.

I meant those several occurrences of a doubled slash in newly added `wx//memoryallocator.h` references (please notice the doubled slash after `wx`). I found it odd as these were the only inclusions using a doubled slash, as opposed to inclusions such as `wx/propdlg.h`.  But if you say it's OK, I believe you.